homeassistant/components/todoist/todo.py

```diff
@@ -44,6 +46,10 @@ def _task_api_data(item: TodoItem) -> dict[str, Any]:
             item_data["due_datetime"] = due.isoformat()
         else:
             item_data["due_date"] = due.isoformat()
+        # In order to not lose any recurrence metadata for the task, we need to
+        # ensure that we send the `due_string` param if the task has it set.
```

Comment:
Can you add a comment here that its ok for this to be stale?

Reply:
Good call, updated!